static/css/frame.scss

```diff
@@ -711,38 +711,38 @@ $bg-white: rgba(255, 255, 255, 0.7);
   @include type-box($small);
 }
 
-.black {
+#text-input.black {
```

Comment:
@punamdahiya Great! Glad you found the fix. This might be nicer with sass nesting, but I'll file a good-first-bug followup to encapsulate the text tool styles more broadly